src/app/src/components/FilterSidebarExtendedSearch.jsx

```diff
@@ -194,110 +198,160 @@ function FilterSidebarExtendedSearch({
                 </div>
             </div>
             <div className="form__field">
```

Comment:
When testing this with an API submission that included only 1 of the 5 extended fields I saw a large amount of white space in the embed sidebar

<img width="766" alt="Screen Shot 2022-04-11 at 3 19 16 PM" src="https://user-images.githubusercontent.com/17363/162844918-641fca7a-4c3f-4bbb-931c-ef5e631edbe1.png">


Consider moving these `<div className="form__field">` tags inside the `ShowOnly` tags so that we don't render extra empty divs.

Reply:
Nice catch! I did see some white spaces but didn't connect it with the empty `div`